rollup.config.ts

```diff
@@ -19,6 +20,9 @@ const externals = [
 
 const extensions = ['.js', '.ts']
 
+// Ignore those polyfills by marking them as external, rollup will treeshake-out the imports for us
+const IGNORE_NODE_POLYFILL = new Set(['os', 'fs'])
```

Comment:
shouldn't it be `IGNORED_NODE_POLYFILLS` ?

Reply:
fixed and amended

Reply:
nice comment update also !